cmd/config-example.env

```diff
@@ -0,0 +1,8 @@
+ALLOWED_ORIGINS=* # or your frontend url
+ALLOW_CREDENTIALS=true
+ADDRESS_ECHO=:5477 #address echo to start
```

Comment:
What those are for?

Reply:
In anticipation of deployment in various contexts and to proactively address potential CORS issues when the application is connected to a browser, I have introduced the following configuration parameters in the file:

ALLOWED_ORIGINS=*: This sets the allowed origins, which can be useful for handling requests from any origin or specifying a specific frontend URL.

ALLOW_CREDENTIALS=true: Enables the forwarding of credentials in case they are required for CORS requests, ensuring greater flexibility in interacting with the frontend.

 ADDRESS_ECHO=:5477 is configured as the address on which the server echo starts